draft-ietf-quic-transport.md

```diff
@@ -1477,11 +1477,14 @@ A final offset is present in both a frame bearing a FIN flag and in a RST_STREAM
 frame.  Upon sending either of these frames for a stream, the endpoint MUST NOT
 send a STREAM frame carrying data beyond the final offset.
 
-An endpoint that receives any frame for this stream after receiving either a FIN
-flag and all stream data preceding it, or a RST_STREAM frame, MUST quietly
-discard the frame, with one exception.  If a STREAM frame carrying data beyond
-the received final offset is received, the endpoint MUST close the connection
-with a QUIC_STREAM_DATA_AFTER_TERMINATION error ({{error-handling}}).
+An endpoint that receives a FIN flag or a RST_STREAM frames knows the final
+offset for this stream. If a STREAM frame carrying data beyond the received
```

Comment:
Personally (and YMMV) I find the new wording here a bit confusing, and I'd keep the previous one for this case (with your addition in the next paragraph).